### PR TITLE
Added rancher 2 helm chart

### PR DESCRIPTION
**Pull request by codejamninja**
Fixes issue #274

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@
 hugo/public
 *.out
 coverage
-bivac
```
Comment (mcanevet) [inline .gitignore]: Could you please propose this change in a separated PR if it is relevant?

File: charts/bivac/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: bivac
+home: https://github.com/camptocamp/bivac
+icon: https://raw.githubusercontent.com/camptocamp/bivac/master/img/bivac_small.png
+version: 0.0.1
+appVersion: latest
+description: 'backup interface for volumes attached to containers'
+keywords:
+  - bivac
+  - bivac
```
Comment (mcanevet) [inline charts/bivac/Chart.yaml]: duplicated keyword

File: charts/bivac/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - backup
+  - bivac
```
Comment (mcanevet) [inline charts/bivac/Chart.yaml]: duplicated keyword

File: charts/bivac/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  - docker
+  - volume
+  - kubernetes
+  - persist
+  - restore
+  - dedup
+  - deduplication
+  - restic
+  - duplicity
```
Comment (mcanevet) [inline charts/bivac/Chart.yaml]: duplicity is not supported anymore

File: charts/bivac/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+sources:
+  - https://github.com/camptocamp/bivac
+maintainers:
+  - name: 'Jam Risser'
```
Comment (mcanevet) [inline charts/bivac/Chart.yaml]: Could you please change this to `Camptocamp`?

File: charts/bivac/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    email: jam@codejam.ninja
```
Comment (mcanevet) [inline charts/bivac/Chart.yaml]: Could you please change this to `info@camptocamp.com`?

File: charts/bivac/templates/services/volback.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```
Comment (mcanevet) [inline charts/bivac/templates/services/volback.yaml]: Please rename this file

File: charts/bivac/values.yaml [hunk 8]
```diff
@@ -0,0 +1,50 @@
+images:
```
Comment (mcanevet) [inline charts/bivac/values.yaml]: It looks like common pattern on github.com/helm/charts id to specify image repository and tag as `image.repository` and `image.tag`. I think we should use the same pattern.

File: charts/bivac/values.yaml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  bivac:
+    repository: camptocamp/bivac
+    tag: 2.0.0
+    pullPolicy: IfNotPresent
+
+config:
```
Comment (mcanevet) [inline charts/bivac/values.yaml]: Maybe we should just expose a `bivac.env: {}` parameter to have maximum flexibility.

**Comment by mcanevet** [comment]
@codejamninja could you please move this to `contrib/rancher2`?

**Comment by codejamninja** [comment]
As a folder `contrib/rancher2` or branch `contrib/rancher2`?

**Comment by mcanevet** [comment]
Folder `contrib/rancher2`

**Comment by codejamninja** [comment]
I think maybe this should be a regular helm chart and I'll submit a rancher chart to the rancher charts repo.

**Pull request closed without merging**